sgx_tstd/src/sys/fd.rs

```diff
@@ -205,13 +171,13 @@ impl FileDesc {
                         FileDesc::new(fd)
                     });
                 }
-                Err(ref e) if e.raw_os_error() == Some(libc::EINVAL) => {
+                Err(ref e) if e.equal_to_os_error(libc::EINVAL) => {
                     TRY_CLOEXEC.store(false, Ordering::Relaxed);
```

Comment:
not sure if `SeqCst` is better. @volcano0dr

Reply:
same to line 162

Reply:
I think Relaxed is okay.